game/src/sandbox/dashboards/table.rs

```diff
@@ -0,0 +1,261 @@
+use crate::app::App;
+use abstutil::prettyprint_usize;
+use geom::Polygon;
+use widgetry::{Btn, EventCtx, GeomBatch, Line, Panel, Text, TextExt, Widget};
+
+// TODO With a bit more detangling from App, this could be lifted to widgetry!
```

Comment:
:+1: - yeah, seems like app is only used for the color scheme?

Reply:
Need to detangle some lifetimes, I'll attempt later